html/pfappserver/root/static.alt/src/globals/pfValidators.js

```diff
@@ -161,3 +242,144 @@ export const userNotExists = (value, component) => {
     return true
   })
 }
+
+/**
+ * Field functions
+ *
+ * For use with pfFormSortableField component.
+ * Used to validate |type| fields with immediate siblings.
+ * All functions ignore self.
+**/
+
+// Limit the count of sibling field |type|s
+export const limitSiblingFieldTypes = (limit) => {
+  return (0, _common.withParams)({
+    type: 'limitSiblingFieldTypes',
+    limit: limit
+  }, function (value, field) {
+    let count = 0
+    const { id, parent, params } = idParentParamsFromV(this.$v, field)
+    if (params) {
+      // iterate through all params
+      for (let i = 0; i < params.length; i++) {
```

Comment:
We could add a condition to return sooner if the count is bigger than the limit.